asminc/time.inc

```diff
@@ -58,6 +58,15 @@
 .endstruct
 
 
+;------------------------------------------------------------------------------
+; ProDOS date/time format
+.ifdef __APPLE2__
+.struct prodos_datetime
+        date        .word
+        time        .word
+.endstruct
+.endif
+
```

Comment:
I don't like this section here. Maybe mli.inc or <do you have a better idea?>

Reply:
Same as for the .h files: putting it elsewhere could be possible, but not in apple2.inc either, or we'd have to include it there, too. Maybe a specific file asminc/prodos_datetime.inc ?

Reply:
I don't agree that it is the same as for the .h files: The .h files include other .h files "where necessary", but .inc files never include other .inc files - at least afaik. It's up to the user to know what to include in which order.

Given that there's no code using this .struct at all I propose to "simply" go without. In stat.inc we just write:
```
.struct stat
        ...
        st_blocks         .word
        st_mod_date       .word
        st_mod_time       .word
        st_create_date    .word
        st_create_time    .word
```